app/webpacker/src/stylesheets/navbar.scss

```diff
@@ -3,9 +3,8 @@
 //  See the COPYING file at the top-level directory or at
 //  https://github.com/puzzle/cryptopus.
 
-
-.navbar-inverse{
-  width:100vw !important;
+.navbar-inverse {
```

Comment:
Not sure where the problem stems from, but when shortening the window in width, the navbar at some point is does not cover 100% of the window. Setting width: 100% seems to fix it.